src/js/components/conversation.jsx

```diff
@@ -53,7 +53,7 @@ export class ConversationComponent extends Component {
     }
 
     render() {
-        const messages = this.props.conversation.messages.map((message) => <MessageComponent key={ message._id } {...message} />);
+        const messages = this.props.conversation.messages.map((message, index) => <MessageComponent key={ index } onLoad={this.scrollToBottom} {...message} />);
```

Comment:
of we scroll for each message `onLoad` now, do we still need the componentDidMount and componentDidUpdate triggers as well?